packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../../../../shared/common_widgets.dart';
+import '../controller/diff_pane_controller.dart';
+
+class SnapshotControlPane extends StatelessWidget {
+  const SnapshotControlPane({Key? key, required this.controller})
+      : super(key: key);
+
+  final DiffPaneController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<bool>(
+      valueListenable: controller.isProcessing,
+      builder: (_, isProcessing, __) => Row(
+        children: [
+          ToolbarAction(
+            icon: Icons.clear,
+            tooltip: 'Delete snapshot',
```

Comment:
instead of having this delete control on the snapshot itself, what about having a little trash can icon that appears over a snapshot list item when hovered? This would be make it more clear that the delete action actually takes a snapshot out of the list completely, and doesn't just close the table view.

Reply:
Yes, it would be great and I tried this path. I did not find a simple way to highlight 'x' on hover.
I would leave it to fix-it-friday.